src/usecases/retrieveVisits.js

```diff
@@ -1,7 +1,9 @@
 const retrieveVisits = ({ getDb }) => async ({ wardId }) => {
   const db = await getDb();
   try {
-    let query = `SELECT * FROM scheduled_calls_table WHERE ward_id = $1 ORDER BY call_time ASC`;
+    let query = `SELECT * FROM scheduled_calls_table 
+                 WHERE ward_id = $1 AND call_time >= NOW() AND status = 'scheduled' 
```

Comment:
We filter the visits by callTime in the `AccordionVisits` component (using 3 filter helpers). As such we don't need to check for visits that have a call time in the future (`call_time >= NOW()`). This query will prevent us from showing visits from the last 12 hours (one of the filters on the Ward visits page)

Reply:
This change is also the reason the e2e tests are failing. A visit is scheduled with a call_time of now, and then the test runs a second after and filters out these 'past' visit.

Reply:
I've removed the time constraint from that query and reviewed all usage without finding any negative consequences.